docs/extensions/engines/spark/lineage.md

```diff
@@ -207,3 +208,23 @@ spark.sparkContext.addSparkListener(new SparkListener {
 #### Get Lineage Events from Kyuubi EventHandler
 
 When using the `KYUUBI_EVENT` dispatcher, the lineage events will be sent to the Kyuubi `EventBus`. Refer to [Kyuubi Event Handler](../../server/events) to handle kyuubi events.
+
+#### Ingest Lineage Entities to Apache Atlas
+
+The lineage entities can be ingested into [Apache Atlas](https://atlas.apache.org/) using the `ATLAS` dispatcher.
+
+Extra works:
+
++ The least transitive dependencies needed, which are under `./extensions/spark/kyuubi-spark-authz/target/scala-${scala.binary.version}/jars`
++ Use `spark.files` to specify the `atlas-application.properties` configuration file for Atlas
+
+Atlas Client configurations (Configure in `atlas-application.properties` or passed in `spark.atlas.` prefix):
+
+|         Name          |  Default Value  |                      Description                      | Since |
+|-----------------------|-----------------|-------------------------------------------------------|-------|
+| atlas.rest.address    | localhost:21000 | The rest endpoint url for the Atlas server            | 1.8.0 |
+| atlas.client.type     | rest            | The client type (currently only supports rest)        | 1.8.0 |
+| atlas.client.username | None            | The client username                                   | 1.8.0 |
+| atlas.client.password | None            | The client password                                   | 1.8.0 |
```

Comment:
The default value for these two configs should be "<undefined>" (the same as kyuubi configs),  while None is a scala type?

Reply:
> The default value for these two configs should be "" (the same as kyuubi configs), while None is a scala type?

There are also some 'none' in kyuubi docs, like rules.md and settings.md. I prefer to use none, which means no default value instead of an empty string.